src/helpers/defaultTestProps.js

```diff
@@ -0,0 +1,7 @@
+// @flow
```

Comment:
is this file necessary? it's used only once

if we need the same in multiple test files, let's hide it in some `__tests__` folder

Reply:
I would say this was a premature optimization 😅